lib/ansible/modules/windows/win_regedit.ps1

```diff
@@ -21,313 +22,317 @@
 
 $ErrorActionPreference = "Stop"
 
-$params = Parse-Args $args -supports_check_mode $true
+$params = Parse-Args -arguments $args -supports_check_mode $true
 $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
-$diff_support = Get-AnsibleParam -obj $params -name "_ansible_diff" -type "bool" -default $false
+$diff_mode = Get-AnsibleParam -obj $params -name "_ansible_diff" -type "bool" -default $false
```

Comment:
I understand that, I liked diff_mode more as to me it indicates whether we are in diff mode or not and this is just the way I've been doing it in the modules. Does it really matter too much what this is set as?